sonar-plugin-api/src/main/java/org/sonar/api/test/MutableTestCase.java

```diff
@@ -38,5 +38,11 @@
   @Deprecated
   MutableTestCase setType(@Nullable String s);
 
+  /**
+   * @deprecated since 5.2. Use {@link #setCoverageBlock(InputFile, List)}
+   */
+  @Deprecated
   MutableTestCase setCoverageBlock(Testable testable, List<Integer> lines);
```

Comment:
![New](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/new.png)![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png)Do not forget to remove this deprecated code someday.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)